scripts/operator/pr_resources.py

```diff
@@ -3,26 +3,24 @@
 content = ''' Signed-off-by: Ivan Stanev istanev93@gmail.com
```

Comment:
Was changing the file permissions `100755 → 100644` intentional?

Reply:
no, how do I / do I need to change it back?

Reply:
`chmod +x scripts/operator/pr_resources.py`

Reply:
👍 done